.github/workflows/release_prep.sh

```diff
@@ -40,7 +43,7 @@ http_archive(
     name = "aspect_rules_ts",
     sha256 = "${SHA}",
     strip_prefix = "${PREFIX}",
-    url = "https://github.com/aspect-build/rules_ts/archive/refs/tags/${TAG}.tar.gz",
+    url = "https://github.com/aspect-build/rules_js/releases/download/${TAG}/${ARCHIVE}",
```

Comment:
typo?

Reply:
intentional. we're switching to release artifacts now for all of our rule sets

Reply:
ohh. you mean the rules_js bit.. yes, thanks for catching that.